Tools/BuildAndPackage.ps1

```diff
@@ -14,18 +26,33 @@ $slnFile = Join-Path $projectRoot "gcloud-powershell.sln"
 $binDir = Join-Path $projectRoot "Google.PowerShell\bin\"
 $debugDir = Join-Path $binDir "Debug"
 
+
```

Comment:
Unnecessary vertical whitespace.

In general, avoid having two successive lines of vertical whitespace except in very rare circumstances. (The JavaScript style guide mandates it for closure classes, and IIRC, Python classes.)